public/src/client/topic/diffs.js

```diff
@@ -23,64 +61,50 @@ define('forum/topic/diffs', ['api', 'forum/topic/images'], function (api) {
 				}),
 				numDiffs: data.timestamps.length,
 				editable: data.editable,
+				deletable: data.deletable,
 			}, function (html) {
-				var modal = bootbox.dialog({
-					title: '[[topic:diffs.title]]',
-					message: html,
-					size: 'large',
-				});
+				const modalExists = !!modal;
+				if (modalExists) {
+					modal.find('.modal-body').html(html);
+				} else {
+					modal = bootbox.dialog({ title: '[[topic:diffs.title]]', message: html, size: 'large' });
+				}
```

Comment:
I am not sold on this, I will propose an alternative.

Reply:
Number of diffs was also required to be updated, so I went with that ugly solution. Pushed a better one. :)